sensor/kubernetes/fake/fake.go

```diff
@@ -123,7 +125,13 @@ func NewWorkloadManager(config *WorkloadManagerConfig) *WorkloadManager {
 		log.Panicf("could not unmarshal workload from file due to error (%v): %s", err, data)
 	}
 
+	db, err := pebble.Open("/var/cache/stackrox/pebble.db", &pebble.Options{})
+	if err != nil {
+		log.Panicf("could not open id storage")
+	}
```

Comment:
Can we make this configurable? I think local-sensor will fail to start (unless run as root) if we try to start it with this path.

Reply:
oh yeah, good point

Reply:
Done, made it an env var that we apply to the scripts but not to local

Reply:
Awesome! 👍